tests/tcp-rst-unacked-stream-12/test.yaml

```diff
@@ -0,0 +1,26 @@
+requires:
+  min-version: 8
+
+args:
+  - -k none
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: fileinfo
+        fileinfo.sha256: 2a6d1d2d85129cf9e84290a94e7b4d7cfe09d80c47a899dbc04cc61cc737c5a4
+  - filter:
+      count: 1
+      match:
+        event_type: fileinfo
+  - filter:
+      count: 1
+      match:
+        event_type: http
+  - filter:
+      count: 1
+      match:
+        event_type: stats
+        stats.app_layer.tx.http: 1
+          #stats.app_layer.flow.http: 1
```

Comment:
nit duplicate

Reply:
I've left it there because ideally we uncomment it, but sadly the flow app-layer counting has issues and so it won't work

Reply:
fair enough, your comment in the PR is better than the comment in the test.yaml ;-)